pkg/eve/networks.go

```diff
@@ -121,8 +126,10 @@ func (ctx *State) NetList() error {
 		return netInstStatesSlice[i].Name < netInstStatesSlice[j].Name
 	})
 	for _, el := range netInstStatesSlice {
-		if _, err := fmt.Fprintln(w, el.toString()); err != nil {
-			return err
+		if !el.deleted {
```

Comment:
We have this check in `func (ctx *State) Networks() []*NetInstState`.

Reply:
Ah but that isn't consistent with other objects such as volumes. Confusing.

Reply:
Fixed